06-building-a-program-start-to-finish/solving-the-problem/output/inverted-index.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * An inverted-index class
+ * @class
+ */
+class InvertedIndex {
+  /**
+  * class constructor
+  * @constructor
+  */
+  constructor() {
+    this.allIndex = {};
+    this.allFiles = {};
+    this.allLength = {};
+    this.allFilesTitle = [];
+    this.validatedFile = {};
+  }
+
+  /**
+   * Validate Method
+   * @param {Object} validateFile an object to be validated
+   * @returns {Object} file that has been validated
+   */
+  validateFile(validateFile) {
+    if (validateFile.title !== undefined && validateFile.text !== undefined) {
+      return true;
+    }
+    return false;
+  }
+
+  /**
+   * sanitizing objects
+   * @param {Object} BookObject receives json object
+   * @return {Array} returns an array of names
+   */
+  sanitizingObj(BookObject) {
```

Comment:
`sanitizingObj` Nope!

`BookObject` Nope!

Please ensure you fix all the issues I pointed out earlier.